### ISSUE TITLE
Fine tune segformer on custom dataset

### ISSUE THREAD

**Issue by driggeraaliya94** [open-mmlab/mmsegmentation#1820]
I want to fine tune segformer on custom dataset, which is exported via "label studio" in coco format. How can I fine tune segformer with the coco format dataset.

**Comment by MengzhangLI** [comment]
We do not have a plan for finetune model. The only potential way is using some checkpoints we provided as pretrained model, such as COCO-STUFF dataset, you can have a try.

**Comment by driggeraaliya94** [comment]
Thank you, MengzhangLI for the quick turn around. Although, I still have a query related to the retraining on the custom dataset.
Can you please suggest me the dataset formats which can be easily used for training the pretrained models.

**Comment by MengzhangLI** [comment]
Vaihingen dataset PR: https://github.com/open-mmlab/mmsegmentation/pull/1171

Unofficial PR doc: https://github.com/MengzhangLI/mmsegmentation/blob/add_doc_customization_dataset/docs/en/tutorials/customize_datasets.md
